cluster/examples/kubernetes/ceph/common-external.yaml

```diff
@@ -35,7 +35,7 @@ roleRef:
 subjects:
```

Comment:
A tag could be added to the namespace above on line 24.

Reply:
I was considering this... I originally had this the other namespaces here as `namespace:cluster` but removed them because it is for an external cluster. I could use `namespace:cluster` or `namespace:external-cluster`.... 

Maybe `namespace:cluster` is best since the user's desired `cluster` may be external or non-external.

Reply:
Using a different tag for the external cluster namespace was my initial impression as well, but I wonder actually if it makes more sense to keep it as `namespace:cluster`. An external cluster is the same type of CR as a local cluster. The storage classes will also need to correspond to that same namespace. And if someone wants to replace it with a different namespace for the external cluster, they can just key off the `rook-ceph-external` name in the example.